cmd/tailscale/cli/switch.go

```diff
@@ -42,12 +44,22 @@ func listProfiles(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
+	tw := tabwriter.NewWriter(os.Stdout, 2, 2, 2, ' ', 0)
+	defer tw.Flush()
+	insertRow := func(vals ...string) {
+		fmt.Fprintln(tw, strings.Join(vals, "\t"))
+	}
+	insertRow("ID", "Tailnet", "Account")
```

Comment:
If we're updating the list here, I think we should update the ID that switching profiles accepts in the same change. Could get the localbackend changes through first then follow up with all of the CLI updates

Reply:
> I think we should update the ID that switching profiles accepts in the same change
I think this PR does this? Or did I miss something 🙈

Reply:
oooh yes i missed that part! sorry!

Reply:
Is there anywhere we should update help text for this command about what to use when picking a profile?

Reply:
Will add it to the "switch" CLI docs